src/mongoModelsRepo/tests/createUpdateIdentifierPersona/createUpdateIdentifierPersonaRetry.test.ts

```diff
@@ -19,19 +19,16 @@ import createUpdateIdentifierPersona from '../../createUpdateIdentifierPersona';
 import createIdentifier from '../../utils/createIdentifier';
 
 describe('createUpdateIdentifierPersona mongo retry', () => {
-
   // Only test mongo repo
   /* istanbul ignore next */
   if (config.repoFactory.modelsRepoName !== 'mongo') {
     return;
   }
 
-  let serviceConfig: ServiceConfig; // tslint:disable-line:no-let
-  let theService: Service; // tslint:disable-line:no-let
-
+  let serviceConfig: ServiceConfig; let theService: Service;
```

Comment:
@crazy-grizzly think this is the same thing as before.
```suggestion
  let serviceConfig: ServiceConfig;
  let theService: Service;
```

Reply:
Oh man, so this is ESLint autofix did this 😕